Source/SwiftLintFramework/Rules/Idiomatic/NoMagicNumbersRule.swift

```diff
@@ -92,3 +113,25 @@ private extension TokenSyntax {
             && grandparent.as(PrefixOperatorExprSyntax.self)?.parent?.is(InitializerClauseSyntax.self) != true
     }
 }
+
+private extension ExprSyntaxProtocol {
+    func isMemberOfATestClass(_ testParentClasses: Set<String>) -> Bool {
+        var parent = parent
+        while parent != nil {
+            if
+                let classDecl = parent?.as(ClassDeclSyntax.self),
+                classDecl.isTestClass(testParentClasses: testParentClasses)
+            {
+                return true
+            }
+            parent = parent?.parent
+        }
+        return false
+    }
+}
+
+private extension ClassDeclSyntax {
+    func isTestClass(testParentClasses: Set<String>) -> Bool {
```

Comment:
The `isXCTestCase` already exists on `ClassDeclSyntax` objects.

Reply:
I should have remembered that :-)